.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: If something isn't working
+labels: ['kind/bug', 'status/triage']
+---
+
+### Describe the Bug?
+
+Provide a clear and concise description of the bug.
+
+### What are the steps to reproduce this issue?
+
+1. …
+2. …
+3. …
+
+### Expected Behavior
+
+What do you think should have happened?
+
+### Actual Behavior
+
+What actually happened? Please provide screenshots if applicable.
+
+### Additional information
+
+- Which devices are affected? ie Desktop, Table, Phone
+- Which browsers are affected? ie Chrome, Firefox, Safari, Edge
+- Which version are you using? (We only support the last 2 major versions.)
```

Comment:
Helm doesn't support any version. In fact you can deploy any image with this chart. How is this relevant?